ies.md

```diff
@@ -235,13 +237,121 @@ Otherwise, things to watch out for in the UML are:
+</ul>
+8. Changes to <b>Asset </b>pattern
+<ul>
+	<li><b>storedIn </b>deleted. Use <b>inLocation </b>instead.</li>
+</ul>
+9. Changes to <b>Communications Device</b> pattern:
+<ul>
+	<li>Removal of <b>installedSoftware</b>, replaced with <b>InstanceOfSoftware </b>which can be associated as being installed using <b>isPartOf.</b></li>
+	<li>Removed <b>ModelOfDevice </b>and <b>ClassOfDevice</b>. Instantiations of <b>ModelOfDevice </b>to be done instead using subClassOf <b>Device.</b></li>
+	<li>Linked Device to <b>PartNumber </b>using <b>isIdentifiedBy</b></li>
+</ul>
+10. Changes to <b>Period of Time</b> pattern:
+<ul>
+	<li><b>ParticularPeriod</b> URI pattern now mandated to be non-punctuated encoding (20070118T153000Z).  This avoids the use of escape characters in the URI. NOTE:<b> </b>the literal for <b>iso8601PeriodRepresentation </b>remains punctuated.</li>
+	<li><b>ParticularPeriod</b> mandated to be in UTC / Zulu time</li>
+	<li><b>Period of Time</b> diagram changed to reflect changes to URI encoding</li>
+	<li>Updates to all examples to include new encoding</li>
+</ul>
+11. Changes to <b>Disposition</b> pattern:
+<ul>
+	<li>Correction to <b>allHaveDisposition</b> rdfs:Domain - fixed to <b>ClassOfElement </b>not <b>Element</b></li>
+</ul>
+12. Changes to <b>Amount of Money </b>pattern:
+<ul>
+	<li>Currency identifier correct to <b>ISO4217Code </b>not <b>ISO639-3Code</b> (country code)</li>
+</ul>
+13. Changes to <b>Online </b>and <b>Communication </b>patterns covering <b>Message</b>. <b>Message </b>used to inherit from both <b>OnlineEvent </b>and <b>Communication </b>which didn't make sense considering <b>SMS </b>is a subtype of <b>Message</b>. Changes included:
+<ul>
+	<li>Deleted subclass relation between <b>Message </b>and <b>OnlineContentEvent</b></li>
+	<li>Added <b>OnlineMessage </b>class with the definition of "A Message that was sent Online."</li>
+	<li>Added <b>OnlineMessage </b>to the <b>OnlineEvent </b>diagram</li>
+	<li>Made <b>OnlineMessage </b>a subclass of <b>OnlineContentEvent </b>and Message</li>
+	<li>Added Communication class to <b>OnlineEvent </b>diagram</li>
+	<li>Changed <b>Message </b>definition from "A Communication or OnlineContentEvent where a message is sent" to "A Communication where a message is sent"</li>
+</ul>
+14. IES 4.2 had ClassOfElements and subProperties of rdf:type which encouraged extending the IES classes via ClassOfElements hierarchy rather than the Elements hierarchy. IES 4.3 prunes some of classes (breaking changes) to discourage this behaviour and encourage one approach of extending IES. This approach is documented in "Extending IES4 2024-03-v1.0 O.pdf". Pruned classes and properties include:
+<ul>
+	<li>From <b>Authorisation pattern - AuthorisationEventClass </b>deleted � just use subclasses of <b>AuthorisationRequest </b>or <b>GrantOfAuthority</b>. Also <b>requestedActivity</b>, <b>grantedActivityType </b>and <b>allAuthorisedAgainst</b> deleted</li>
```

Comment:
some kind of weir character in this line, guessing it is something gone wrong with a copy paste